README.md

```diff
@@ -55,6 +55,26 @@ zpm update
 
 ```
 
+To remove plugin:
+
+```sh
+zpm remove user/plugin
```

Comment:
Not actually necessary but to keep a unified syntax on the README.md and in general, it should quote the plugin name. ie: `zpm remove "user/plugin"`